lib/dialects/abstract/query.js

```diff
@@ -19,8 +19,7 @@ class AbstractQuery {
     this.options = {
       plain: false,
       raw: false,
-      // eslint-disable-next-line no-console
-      logging: console.log,
+      logging: console.debug,
```

Comment:
What would this change mean effectively for the end users?

Reply:
Also holds for the other instances

Reply:
`console.debug` is an alias for `console.log`

I'm not sure where to document that but I've configured eslint to forbid `console.log` while still allowing every other console methods.
The idea behind it is that we often use `console.log` to debug something. With this rule, we'll be warned that we forgot to remove throwaway logging.
`console.debug` is used to say "I am logging this deliberately."

For the same reason `debugger` is also forbidden by eslint.

People can still use them during development (encouraged even), but once they commit it should be removed :)

Reply:
I thought that console.debug would only show if you changed some configuration so I'm not sure if all end users have that set up so that they will see these outputs. So not sure what to do with that to be honest

Reply:
If you use another logger but not the base console.debug, it's just an alias :) https://nodejs.org/api/console.html#consoledebugdata-args

Reply:
Fun fact, console.info is also just an alias https://nodejs.org/api/console.html#consoleinfodata-args

Reply:
Ah, that's good to know. Reasonable change then